.travis.yml

```diff
@@ -1,29 +1,34 @@
 language: php
 
-php:
-  - '7.0'
-  - '7.1'
-  - nightly
+sudo: false
+
+git:
+    depth: 1
 
 matrix:
-  fast_finish: true
-  allow_failures:
-    - php: nightly
+    fast_finish: true
+    include:
+        - php: 7.0
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
```

Comment:
yeah guess what I tried before this line : P
I still consider this a compoesr ~bug~ DX issue